README.md

```diff
@@ -73,6 +73,21 @@ const nwcClient = await nwc.NWCClient.fromAuthorizationUrl(
 
 The same options can be provided to getAuthorizationUrl() as fromAuthorizationUrl() - see [Manual Auth example](./examples/nwc/client/auth_manual.html)
 
+#### `static newClientFromAuthorizationUrl()`
+
+Initialized a new `NWCClient` instance but generates a new random secret. The pubkey of that secret then needs to be authorized by the user (this can be initiated by redirecting the user to the `getAuthorizationUrl()` URL or calling `fromAuthorizationUrl()` to open an authorization popup.
```

Comment:
typo: Initialize`s` + heading should be `fromAuthorizationUrl`

But wait isn't this same as above? (L59-74)